release/_releases_json.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
 
+function _add_new_version {
+	lc_log INFO "Add ${LIFERAY_RELEASE_VERSION} version to supported-${LIFERAY_RELEASE_PRODUCT_NAME}-versions.txt"
+
+	if [ -n "${LIFERAY_RELEASE_PRODUCT_NAME}" ]
+	then
+		echo -en "\n${LIFERAY_RELEASE_VERSION}" >> "${_RELEASE_ROOT_DIR}"/supported-"${LIFERAY_RELEASE_PRODUCT_NAME}"-versions.txt
```

Comment:
@Wykenz, checking this code, I'm realizing that ideally we should validate if the content of the variable `LIFERAY_RELEASE_VERSION` is valid, otherwise we may be introducing incorrect information into the file `supported-dxp-versions.txt`. In light of this, how about you just validate if the content of the variable `LIFERAY_RELEASE_VERSION` is already inserted into the file `supported-dxp-versions.txt`?

If it's not there then we should display an error in the log and also we shouldn't regenerate the file `releases.json`, does it make sense to you?

Thanks